src/shared/components/community/sidebar.tsx

```diff
@@ -132,8 +132,8 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
     const myUSerInfo = UserService.Instance.myUserInfo;
     const { name, actor_id } = this.props.community_view.community;
     return (
-      <div>
-        <div className="card border-secondary mb-3">
+      <div id="sidebarContainer">
```

Comment:
Maybe aside would be better than div for this?

Reply:
This is resolved in the latest commit. Here's the structure of the sidebar on the homepage:

![image](https://github.com/LemmyNet/lemmy-ui/assets/3112763/85ac2c50-0198-43b3-94b5-914706991000)

And then on a post:

![image](https://github.com/LemmyNet/lemmy-ui/assets/3112763/a021f72d-5e30-46bd-85c0-930b97a80cc6)